src/services/org.services.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  OrgPaymentDetails
+} from '@/models/Organization'
+
+import { AxiosResponse } from 'axios'
+import ConfigHelper from '@/util/config-helper'
+import { axios } from '@/util/http-util'
+
+export default class OrgService {
+  static async getOrgPayments (orgId: number): Promise<AxiosResponse<OrgPaymentDetails>> {
+    return axios.get(`${ConfigHelper.getPayAPIURL()}/accounts/${orgId}`)
+  }
+}
```

Comment:
Please rename this from orgService -> payService or refactor into existing pay service

This is calling `pay-api`